lib/awful/screen.lua

```diff
@@ -626,10 +753,187 @@ end
 -- @signal request::wallpaper
 -- @tparam screen s The screen object.
 
+--- When a new physical viewport has been added.
```

Comment:
What is a viewport? This seems to be a new concept and I fail to find any documentation about it.

Reply:
modified to avoid using the "viewport" word in the public doc.